### PR TITLE
 installed python tf2_geometry_msgs

### PR DESCRIPTION
**Pull request by ahcorde**
- installed python tf2_geometry_msgs
 - Added TODO to install test once this PR https://github.com/ros2/orocos_kinematics_dynamics/pull/4 is merged

### CODE REVIEW

File: tf2_geometry_msgs/src/tf2_geometry_msgs/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-from tf2_geometry_msgs import *
+from .tf2_geometry_msgs import *
```
Comment (clalancette) [inline tf2_geometry_msgs/src/tf2_geometry_msgs/__init__.py:1]: I'm not opposed to this change, but I'm surprised it is needed.  Is it?

**Review by clalancette — COMMENTED** [review]
I have one question, but otherwise this looks pretty good to me.

File: tf2_geometry_msgs/src/tf2_geometry_msgs/__init__.py (hunk 1 above)
Comment (ahcorde) [reply tf2_geometry_msgs/src/tf2_geometry_msgs/__init__.py:1]: there is code inside `tf2_geometry_msgs.py` that needs to run when the module is imported. I didn't find another way to do it

Comment (clalancette) [reply tf2_geometry_msgs/src/tf2_geometry_msgs/__init__.py:1]: All right, that's fine.  I'll approve.

**Review by clalancette — APPROVED** [review]

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/8910/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/4692/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/7292/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/8827/)

**Pull request merged**